op-challenger/game/fault/contracts/faultdisputegame.go

```diff
@@ -279,6 +279,11 @@ func (f *FaultDisputeGameContract) resolveCall() *batching.ContractCall {
 	return f.contract.Call(methodResolve)
 }
 
+// Inner returns the inner [*batching.BoundContract].
+func (f *FaultDisputeGameContract) Inner() *batching.BoundContract {
+	return f.contract
+}
```

Comment:
We shouldn't break encapsulation by exposing this.  Just add methods to this contract to support the calls we need to make.